great-idea/css/index.css

```diff
@@ -62,3 +62,141 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+
+/* ~~~~~~~~~~~~~~~ My Code ~~~~~~~~~~~~~~~ */
+
+.container {
+	display: flex;
+	
+	margin-top: 20px;
+	margin-left: 40px;
+	margin-right: 40px;
+	margin-bottom: 60px;
```

Comment:
You can make your code more efficient by using the shorthand `margin` property: https://www.w3schools.com/css/css_margin.asp